docs/tutorial.rst

```diff
@@ -962,37 +954,36 @@ Let's verify that we have both neutral and deleterious mutations in the resultin
         f"{len(mut_info) - num_neutral} nonneutral mutations."
     )
 
+    # There are 323 neutral mutations, and 420 nonneutral mutations.
+
+
+.. _sec_tute_selection_single_gene:
 
 2. Simulating selection in a single gene
 ----------------------------------------
 
-Next, we'll simulate a 1kb gene flanked by 1kb neutral regions.
-Within genes, 30% of the total influx of mutations are neutral and 70% are deleterious,
-with the DFE again from Kim et al., using the HomSap/OutOfAfrica_4G09 demographic model.
-Within the gene, the neutral mutations are added by msprime (as above),
-and outside the gene, all mutations are neutral and all are added with msprime.
+Next, we'll simulate a 10kb gene flanked by 10kb neutral regions,
+by specifying a particular interval to apply the HomSap/Gamma_K17 DFE to.
+Contigs come by default covered by a neutral DFE,
+so all we need to do is apply the DFE to the middle region.
```

Comment:
Should we use more words to describe what it means to apply a DFE on top of the existing neutral one?